orchagent/aclorch.cpp

```diff
@@ -2903,8 +2903,16 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
                 type = table_id == m_mirrorTableId[stage] ? ACL_TABLE_MIRROR : ACL_TABLE_MIRRORV6;
             }
 
-
-            newRule = AclRule::makeShared(type, this, m_mirrorOrch, m_dTelOrch, rule_id, table_id, t);
+            try
+            {
+                newRule = AclRule::makeShared(type, this, m_mirrorOrch, m_dTelOrch, rule_id, table_id, t);
+            }
+            catch (exception &e)
```

Comment:
>exception [](start = 19, length = 9)

`exception` type is too generic and possibly hide code bug. Could you list all possible excpetion types?
This comment is also applicable to master branch. So if you found better solution, let's start a PR on master first.